scripts/integration-helpers/framework.ts

```diff
@@ -263,6 +263,21 @@ export class TriggerEndToEndTest {
     return fetch(url);
   }
 
+  invokeCallableFunction(
+    name: string,
+    body: Record<string, unknown>,
+    zone = FIREBASE_PROJECT_ZONE
+  ): Promise<Response> {
+    const url = `http://localhost:${[this.functionsEmulatorPort, this.project, zone, name].join(
```

Comment:
This is real strange, just given how URLs should be thought of. Consider pulling the port out of the path:

```suggestion
    const url = `http://localhost:${this.functionsEmulatorPort}/${this.project, zone, name].join(
```